lib/logflare_web/live/search_live/logs_search_lv.ex

```diff
@@ -120,7 +122,8 @@ defmodule LogflareWeb.Source.SearchLV do
     socket =
       with {:ok, lql_rules} <-
              Lql.decode(qs, get_bigquery_schema(source)),
-           lql_rules = Lql.Rules.put_new_chart_rule(lql_rules, Lql.Rules.default_chart_rule()),
+           lql_rules = Rules.put_new_chart_rule(lql_rules, Rules.default_chart_rule()),
+           lql_rules = ensure_from_rule(lql_rules, source),
```

Comment:
don't think we need `ensure_from_rule/2` in this situation, since search ui is already restricted to 1 source so having a from rule would be redundant. 
also think that adding in a `from:` rule to user LQL as of now might break queries when we do a gradual rollout as clusters on previous versions would be lagging behind and would not know of what a `from` rule is.